tools/catalog-report.py

```diff
@@ -56,7 +56,7 @@ def log_entry(x):
 
 for i, x in enumerate(j):
     if i % freq == 0:
-        print(i, end='\r')
+        print('{}'.format(i), end='\r')
```

Comment:
i guess i am not a python guru enough for this, but what precisely does this do? afaics this just changes the type of what is printed here from an integer to a string, but if that's the intention, why not use str()?